### PR TITLE
feat: poll configurations

### PR DESCRIPTION
**Pull request by MartianGreed**

### CODE REVIEW

File: internal/indexer/config.go [hunk 1]
```diff
@@ -26,3 +26,33 @@ func checkConfigChange(ctx context.Context, storage storage.Storage, cfg config.
 		}
 	}
 }
+
+func fetchConfigurations(cr config.ContractRepository, cfgChan chan<- []config.Config) {
+	for {
+		time.Sleep(5 * time.Second)
+		cfgs, err := cr.GetConfigs()
+		if err != nil {
+			log.Error("failed to get configs", "error", err)
```
Comment (tekkac) [inline internal/indexer/config.go:35]: debug here?

Comment (tekkac) [reply internal/indexer/config.go:35]: no get out of loop?

**Review by tekkac — APPROVED** [review]
:loop:

File: internal/indexer/config.go (hunk 1 above)
Comment (MartianGreed) [reply internal/indexer/config.go:35]: could be single error not necessarily repearting

Comment (MartianGreed) [reply internal/indexer/config.go:35]: i.e database timeout while server restarting, might not want to exit yet

**Pull request merged**